likecoin/public/iscn-badge.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * LikeCoin iscn badge
+ *
+ * LikeCoin iscn badge displays the iscn id of the content owner
+ *
+ * @package   LikeCoin
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+/**
+ * Require shared functions
+ */
+require_once dirname( __FILE__ ) . '/../includes/likecoin.php';
+
+/**
+ * Add iscn badge if hashed transaction id exist;
+ *
+ * @param string| $iscn_hash: hashed transaction id;
+ */
+function likecoin_add_iscnbadge( $post, $iscn_hash = '') {
+	$post_id = $post->ID;
+    $iscn_info = get_post_meta( $post_id, LC_ISCN_DEV_INFO, true );
+    $iscn_hash = isset( $iscn_info['iscn_hash'] ) ? $iscn_info['iscn_hash'] : null;
```

Comment:
The input param `iscn_hash` will 100% be overwritten here, not sure if is is suppose to override this line instead